code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -161,6 +163,9 @@
 				new /obj/item/clothing/suit/armor/ascetic(src)
 			if(29)
 				new /obj/item/kitchen/knife/envy(src)
+			if(30)
+				new /mob/living/simple_animal/hostile/poison/bees/toxin(src)
```

Comment:
as funny as this is, you probably shouldn't do it.

Reply:
Ah, but it was so worth a try!

Reply:
It would be so so so so SO funny. But not this item pool. Maybe something else, later.

Reply:
As a fun fact, they already HAVE existed in a different item pool. Went back and forth on removing them there, but based on the fact that I haven't heard a single player discover it yet, I assume it's fine.